lms/static/sass/views/_login-register.scss

```diff
@@ -385,6 +385,10 @@ $sm-btn-linkedin: #0077b5;
             line-height: $third-party-button-height;
             text-align: center;
             color: $white;
+
+            img {
+                width: 100%;
```

Comment:
I'm worried this icon will be too big compared to the Google and Facebook Font Awesome icons. When creating mockups for these stories, I think I just used some styles like `"background-image: url(...); background-size: 24px auto; background-position: center;"` on the `.icon` element (something like that).

Can you create a new image that's just a square white PNG, 100px x 100px, and use that for your screenshots? And enable the Google and Facebook providers too? It will give us a better idea of the sizing.

Just specifying the width could also break the layout if some user uploads a logo that has dimensions like 40px x 70px; it would be awesome if we could handle that case, maybe with a max-height rule.